### PR TITLE
chore: use 0.0.10 to build self

### PR DESCRIPTION
**Pull request by fi3ework**
## Checklist

- [ ] Tests updated (or not required).
- [ ] Documentation updated (or not required).

### CODE REVIEW

**Comment by fi3ework** [comment]
We need to speed up CI, especially benchmark. Considering disable dts when building, we need to pass inlined config to merge with the file config first.

**Comment by Timeless0911** [comment]
The lockfile change seems a little extra, pnpm update may cause resolution again.

**Comment by fi3ework** [comment]
Reverted part of the lockfile.

File: tests/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
   "devDependencies": {
     "@codspeed/vitest-plugin": "^3.1.1",
     "@playwright/test": "1.47.2",
-    "@rsbuild/core": "1.0.7",
+    "@rsbuild/core": "1.0.12",
```
Comment (Timeless0911) [inline tests/package.json:16]: Why this not be bumped by renovate

Comment (Timeless0911) [reply tests/package.json:16]: image
we should modify this and the `e2e/cases` is deprecated now.

**Review by Timeless0911 — APPROVED** [review]

**Pull request merged**